staging/src/k8s.io/kubectl/pkg/cmd/create/create_clusterrolebinding_test.go

```diff
@@ -67,49 +62,48 @@ func TestCreateClusterRoleBinding(t *testing.T) {
 		},
 	}
 
-	tf := cmdtesting.NewTestFactory().WithNamespace("test")
+	var receivedApiRequests int32
+	mockServer := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
That's overly complicated. I'd advise you to either 
1. use simple testing similar to what we have in https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/create/create_job_test.go, or
2. or use a fake client (each group has its own) and use reactors like this https://github.com/kubernetes/kubernetes/blob/0e93436ce1d55e71870b5ced1ffbc0cfb2d47f82/staging/src/k8s.io/kubectl/pkg/drain/drain_test.go#L236

Reply:
done